### PR TITLE
Make location picker pageSize  configurable 

### PR DESCRIPTION
**Pull request by donaldkibet**
### What does this PR do?

1. Makes the location-picker page size configurable
2. Display the correct number of locations when less than the default

### Screenshoot
screencast 2021-03-11 01-00-17

### CODE REVIEW

File: src/config-schema.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        "choose a location.",
+    },
+  },
+  chooseLocationPagesize: {
+    pageSize: {
+      _type: Type.Number,
+      _default: 8,
+      _description: "The number of locations displayed on location picker",
+    },
+  },
```
Comment (brandones) [inline src/config-schema.ts:20]: How about

```suggestion
  chooseLocation: {
    enabled: {
      _type: Type.Boolean,
      _default: true,
      _description:
        "Whether to show a 'Choose Location' screen after login. " +
        "If true, the user will be taken to the loginSuccess URL after they " +
        "choose a location.",
    },
    numLocationsToShow {
      _type: Type.Number,
      _default: 8,
      _description: "The number of locations displayed in the location picker",
    },
  },
```

**Comment by brandones** [comment]
This is cool. @donaldkibet , does the location picker now page when there are lots of results? Or if a large value is configured, will it just make the list scrollable if it is too long for the window?

We should only call it "page size" if it's actually paging. Otherwise it's just "list size" or "number to show" or something like that.

**Comment by donaldkibet** [comment]
@brandones it makes the list scrollable when it's too long for the window. According to the design, the default list to show is 8

**Review by brandones — APPROVED** [review]
Looks good, thanks @donaldkibet !

**Review by FlorianRappl — APPROVED** [review]

**Pull request merged**